app/models/metasploit/credential/pkcs12.rb

```diff
@@ -24,15 +33,42 @@ class Metasploit::Credential::Pkcs12 < Metasploit::Credential::Private
 
   validates :data,
             presence: true
+
   #
   # Method Validations
   #
 
   validate :readable
 
+  #
+  # Class methods
+  #
+
   #
   # Instance Methods
   #
+  #
+
+  # The CA that issued the certificate
+  #
+  # @return [String]
+  def ca
+    metadata['ca']
```

Comment:
Use specific ADCS CA name to allow potential other CA type addition in the future.
```suggestion
  def adcs_ca
    metadata['adcs_ca']
```